pkg/service/rpc/roommanager.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package rpc;
+
+option go_package = "github.com/livekit/livekit/pkg/service/rpc";
+
+import "options.proto";
+import "livekit_internal.proto";
+import "livekit_rtc.proto";
+import "google/protobuf/empty.proto";
+
+service RoomManager {
+  rpc RelaySignal(RelaySignalRequest) returns (RelaySignalResponse) {
+    option (psrpc.options).topics = true;
+    option (psrpc.options).stream = true;
+  };
+  rpc RTCNodeMessage(livekit.RTCNodeMessage) returns (google.protobuf.Empty) {
```

Comment:
I know this PR is already getting pretty large.

I think these could actually expand into RoomService calls so we can return the right responses and error messages. That would solve another headache of mine :)